corpus_test.go

```diff
@@ -244,11 +245,27 @@ func TestCorpusRelated(t *testing.T) {
 	tests := []struct {
 		name     string
 		policy   string
+		entities types.EntityGetter
 		request  cedar.Request
 		decision cedar.Decision
 		reasons  []cedar.PolicyID
 		errors   []cedar.PolicyID
 	}{
+		{
+			"a9fe7e4b20024dc7818a168c67ce312d6e076b93",
+			`forbid(
+			principal,
+			action in [Action::"action",Action::"action"],
+			resource
+			) when {
+			true && (resource.hasTag("A"))
+			};`,
+			types.EntityMap{cedar.NewEntityUID("a", ""): cedar.Entity{Attributes: cedar.NewRecord(cedar.RecordMap{"A": types.False})}},
+			cedar.Request{Principal: cedar.NewEntityUID("a", ""), Action: cedar.NewEntityUID("Action", "action"), Resource: cedar.NewEntityUID("a", "'")},
+			cedar.Deny,
+			nil,
+			nil,
+		},
```

Comment:
I'm not 100% convinced we should check this in. I think it would make sense instead to add a pared down version of this to `TestIsAuthorized()`.

For example, I don't think we actually care about the duplicate `Action::"action"` in the action set. You want to validate some very specific behaviors about `getTag()` and `hasTag()`.

Reply:
Yes, done here: https://github.com/cedar-policy/cedar-go/pull/77/commits/9074d5b00f44be4346c13ac8b7270a09b06c541a.